examples/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.1)
 set(CMAKE_CXX_STANDARD 14)
 
 set(REQUIRED_BOOST_LIBRARIES)
-list(APPEND REQUIRED_BOOST_LIBRARIES "log" "program_options" "system" "serialization")
+list(APPEND REQUIRED_BOOST_LIBRARIES "log" "program_options" "system" "serialization" "date_time")
```

Comment:
I will have to add date_time to .travis.yml  boost date_time library , I only build with set of boost libraries
no action required from you on this item.